network/network.go

```diff
@@ -263,7 +263,7 @@ func (n *Network) Configure(config core.ServerConfig) error {
 }
 
 // emitEvents is called when a payload is added.
-func (n *Network) emitEvents(event dag.Event) (bool, error) {
+func (n *Network) emitEvents(_ context.Context, event dag.Event) (bool, error) {
```

Comment:
Not required, since there's nothing to audit here. Although you could say the audit info should travel with the event to the subscriber (then, a retried event retains the original audit info fields, e.g. the original actor)?

I'll still add the context, since we can replace `context.Background()` with it.

Reply:
> I'll still add the context, since we can replace context.Background() with it.

But the added contexts are not used for cancellation, that would give a bad hint for anyone using it?